canonical/demos.haus
# Improved PR deployment comment

## Done

- Added links and images for the demo and Jenkins URL

## QA

- Check that the demo is running.
- Create a new PR on [webteam](https://github.com/canonical/webteam.canonical.com)
  - I've added the webhook to the repository, so creating any new PR should also trigger a new comment on that PR
- Check that the comment is well formatted, and that the links work
